pkg/apis/servicecatalog/types.go

```diff
@@ -266,10 +266,16 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is set of properties to be
 	// passed to the underlying broker.
 	Parameters *runtime.RawExtension
 
+	// Optional: AlphaSecretParameters is reference to the secret object containing
```

Comment:
@duglin if both can be set at the same time, we'll have to define rules for merging. And I would like us to do that as part of a design discussion/proposal process, and keep out of scope for this PR.